src/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics/AtmosphericDrag.cpp

```diff
@@ -121,10 +134,11 @@ void AtmosphericDrag::applyContribution(
     const Vector3d dragAccelerationSI =
```

Comment:
This `-(0.5 / mass) * dragCoefficient * surfaceArea` should be stored in a private `Real` coefficient at construction time. Then we save 3 unnecessaries operations.

Reply:
Let's do this in a separate MR.